src/adapter.js

```diff
@@ -157,6 +157,7 @@ function getAllSpecNames (topSuite) {
  */
 function KarmaReporter (tc, jasmineEnv) {
   var currentSuite = new SuiteNode()
+  var _Date = Date
```

Comment:
Please add a comment as to why this is happening, otherwise the next one editing this code will remove it :D